ramp.yml

```diff
@@ -12,6 +12,7 @@ capabilities:
     - failover_migrate
     - persistence_rdb
     - persistence_aof
+    - clustering
```

Comment:
@swilly22 @K-Jo should we remove the repeated line?

Reply:
@filipecosta90 you're right, it seems like my mistake on merge